app/models/exhibit_proxy.rb

```diff
@@ -8,13 +8,17 @@ def initialize(exhibit)
     @exhibit = exhibit
   end
 
-  def reindex(log_entry = nil)
+  def reindex(*_args)
```

Comment:
Will the caller send unpredictable args?

Reply:
It's the same signature as whatever is sent to Spotlight::Resource - we just don't use them, so it seemed like it'd make this class churn less to just accept them and not do anything with them.